src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx

```diff
@@ -245,6 +245,14 @@ You can follow these recommended actions to improve and get the most out of your
         You'll get more incidents correlated with your issues. You'll get richer and more actionable context from the first notification. The longer the grace period, the later the notification.
       </td>
     </tr>
+    <tr>
+      <td>
+        Periodically review the [issue feed](/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence/#issue-feed)
+      </td>
+      <td>
+        Scroll through the 'Notified' column to make sure all issues are routed to at least one destination. If no routing is necessary, consider removing the condition as it may be noise.
```

Comment:
```suggestion
        Scroll through the **Notified** column to make sure all issues are routed to at least one destination. If no routing is necessary, consider removing the condition as it may be noise.
```